src/types/index.ts

```diff
@@ -0,0 +1,12 @@
+// TODO(laithyounesy@gmail.com): Types should be moved to each component directory
+
+export type CardData = {
+  title: string;
+  description: string;
+  flowID: string;
+};
+
+export type TwoColumnLayoutProps = {
```

Comment:
Could we consider using a consistent naming convention for shared types? I noticed that the "**Props**" suffix is used for some types, but the "**CardData**" type above does not include it. Alternatively, if we add "**Props**" to the shared types, would it indicate that these types include base props for components(e.g. **React.ReactNode**), rather than just typed props(e.g. **string, number, ..etc**)?

Additionally, I've noticed that the **TwoColumnLayoutProps** type is only used within the t**wo-column** layout component. Considering this, what are your thoughts on moving the **TwoColumnLayoutProps** type definition inside the **two-column** folder?

Reply:
here we are using CardData for both props and other other cases, but in case of the TwoColumn LayoutProps in will always be a prop, 

we should have a clear guide for naming, i totally agree with you!